data/settings_mapping.json

```diff
@@ -178,7 +178,8 @@
             "triforce_count_per_world",
             "triforce_goal_per_world",
             "free_bombchu_drops",
-            "one_item_per_dungeon"
+            "one_item_per_dungeon",
+            "plandomized_locations"
```

Comment:
Does this work correctly? It might make more sense to leave this setting off the GUI and make it presets/CLI only.

Reply:
It's been a while since I wrote this code, but I very specifically remember this being necessary because otherwise the preset value would not be included in the setting string, or something along those lines.  i made a comment about this almost exactly one year ago in dev-public-talk:

```
for the 0 people interested in this issue; if the setting is not included in settings_mapping.json, it won't get picked up by the GUI, even if the setting is not intended to be editable by the GUI.  so adding it to that file and giving the setting a name (that will never be shown) but also keeping the gui_type as None was enough to fix things.
```

https://discord.com/channels/274180765816848384/512048482677424138/1154144353539850412